source/partials/release_notes/_release-17-2-0.md.erb

```diff
@@ -0,0 +1,119 @@
+
+#### Feature Enhancements
+
+* <%= link_to_commit 'c08843512b494ef1c483ebb699c615170dd13cf0', "Template Authorization UI and API enhancements" %>
+
+* No restriction on template view config. All pipeline group admins by default provided view permission to templates. Super admin can restrict this access.
+* Super admin can provide template view access to users/roles.
+* Allow template admins to delete templates via UI and API
+
+<figure>
+  <%= image_tag 'releases/17.2.0/template-permissions.png', alt: 'Template Permissions' %>
+</figure>
+
+#### Bug Fixes
+
+* <%= link_to_commit '7ddf84eaaf813546aa1f8eb684ef9cf3187f6617', 'User search functionality - Search through plugins implementing AuthenticationExtension.', 3107 %>
+* <%= link_to_commit 'a837fc05cca60a9bbb8dcd7d3a3c7b1228a51358', 'Sort resources and environments alphabetically on agents page selector dropdown.', 3134 %>
+* Multiple UI fixes on the old and new themes.
+
+
+#### Security Fixes
+
+* <%= link_to_commit 'ad1a1188d15839ac4a41c91c84c631612e49bf60', 'Sanitize scm messages', 3171 %>
+
+#### Breaking changes
+
+* <%= link_to_commit '8f88269fd8165246820ad1c27685485dd3fe0627', 'Removed support for API based plugins', 3102 %>
+
+  Below is the list of plugins that will stop working in 17.2.0
+
+  **Package plugins**
+
+  * [Maven (Nexus) repository poller](https://github.com/aresok/go-maven-poller)
+  * [Docker Registry Poller - 1](https://github.com/hammerdr/go-docker-registry-poller)
+  * [Docker Registry Poller - 2](https://github.com/decoomanj/gocd-docker-material-poller)
+  * [Puppet Forge repository poller](https://github.com/drrb/go-puppet-forge-poller)
+  * [Debian repository poller](https://github.com/gocd-contrib/deb-repo-poller)
+
+  **Task plugins**
+
+  * [MSBuild Task](https://github.com/akanyer/gocd-msbuild-taskplugin/blob/master/README.md)
+  * [XUnit Converter](https://github.com/gocd-contrib/xunit-converter-task)
+  * [Publish to S3](https://github.com/indix-eng/gocd-s3-artifacts)
+  * [Fetch from S3](https://github.com/indix-eng/gocd-s3-artifacts)
+  * [Powershell Task](https://github.com/manojlds/gocd-powershell-runner/blob/master/README.md)
+  * [RapidDeploy Package Builder](https://github.com/MidVision/go-rapiddeploy/wiki#rapiddeploy-package-repository-plugin)
+  * [RapidDeploy Job Runner](https://github.com/MidVision/go-rapiddeploy/wiki#rapiddeploy-job-runner-task-plugin)
+  * [Maven Task Plugin](https://github.com/ruckc/gocd-maven-plugin)
+  * [Publish to Artifactory](https://github.com/tusharm/go-artifactory-plugin)
+
+
+
+
+
+* <%= link_to_commit 'a6f1fac8231a5ee90aedc0395d6c31cd3a451021', 'Removed Pipeline Config API V2', 3174 %>
+
+  As of release 16.12, version 2 of the <%= link_to_api 'pipeline-config', 'Pipeline Config API' %> is deprecated. Users are encouraged to use Version 3 of the API which is backward compatible.
```

Comment:
can we say -- As of release 16.12, version 2 of the Templates API **was** deprecated. Users are encouraged to use Version 3 of the API which is backward compatible **with Version 2**.